examples/python-udwf.py

```diff
@@ -0,0 +1,270 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
As a side comment: Should we have automated tests of the examples? Nothing fancy; simple execution of the scripts should be enough

Reply:
That's a great idea. We already do it for the tpc-h examples. How about we open an issue to run all the examples in CI?

Reply:
Added #888

Reply:
Thanks!